zipkin-autoconfigure/ui/src/main/java/zipkin/autoconfigure/ui/ZipkinUiProperties.java

```diff
@@ -24,6 +24,8 @@
   private int defaultLookback = (int) TimeUnit.DAYS.toMillis(7);
   private String instrumented = ".*";
   private String logsUrl = null;
+  private float dependencyLowErrorRate = 0.5f; // 50% of calls in error turns line yellow
```

Comment:
Along those lines, should we have some validation that values are within bounds?